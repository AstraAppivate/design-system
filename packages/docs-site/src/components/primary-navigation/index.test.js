import React from 'react'
import renderer from 'react-test-renderer'

import PrimaryNavigation from './index'

describe('Primary Naviation', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<PrimaryNavigation />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
