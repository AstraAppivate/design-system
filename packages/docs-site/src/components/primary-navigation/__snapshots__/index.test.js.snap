// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Primary Naviation renders correctly 1`] = `
<nav
  className="primary-navigation"
>
  <a
    className="primary-navigation__link"
    href="/"
  >
    Home
  </a>
  <a
    className="primary-navigation__link"
    href="/design"
  >
    Design
  </a>
  <a
    className="primary-navigation__link"
    href="/develop"
  >
    Develop
  </a>
  <a
    className="primary-navigation__link"
    href="/contact-us"
  >
    Contact Us
  </a>
</nav>
`;
